src/components/sprite-info/sprite-info.css

```diff
@@ -31,14 +31,12 @@ $form-radius: calc($space / 2);
     align-items: center;
 }
 
-/* @todo: refactor from descendant selector, to a regular class */
 .icon-wrapper {
     display: inline-block;
     box-sizing: content-box;
     width: 1.25rem;
     height: 1.25rem;
     padding: calc($space / 2);
-    cursor: pointer;
```

Comment:
Does this make any visual changes or just refactor redundant code? It doesn't _seem_ to when trying it out, just wanted to check.